tests/console/gdb.pm

```diff
@@ -0,0 +1,83 @@
+# SUSE's openQA tests
+#
+# Copyright (C) 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+# Summary: Basic GDB test. (Breakpoints/backtraces/attaching)
+# Maintainer: apappas@suse.de
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+
+sub wait_serial_or_croak
+{
```

Comment:
I wonder why the tidy checks do not complain about this. To keep in line with the style of the rest of the project would you please put the opening `{` on the line one above?